src/styles/blocks/_stars.scss

```
@@ -0,0 +1,37 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  
+  align-items: center;
+
+  &__container {
+    display: flex;
+  }
+
+  &__container-star {
+    background-image: url("../images/star.svg");
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+  }
```

Use nesting with styles for BEM block elements in scss. Fix it everywhere
```suggestion
  &__container {
    display: flex;
    
    &-star {
      background-image: url("../images/star.svg");
      background-position: center;
      background-repeat: no-repeat;
      width: 16px;
      height: 16px;
    }
  }
```